ern-api-gen/src/languages/JavascriptClientCodegen.ts

```diff
@@ -350,7 +350,7 @@ export default class JavascriptClientCodegen extends DefaultCodegen {
         JavascriptClientCodegen.PROJECT_NAME
       )
     ) {
-      ;(this as any).setProjectName(
+      ; (this as any).setProjectName(
```

Comment:
i didn't do almost any of this formatting.  i think the ern teams hooks or something is formatting files that have changed.  my changes were surgical/focused, just FYI if a reviewer is wondering why there are so many diffs?

Reply:
@cdaringe I'm still not sure what exactly caused these lines to be reformatted. Running Prettier locally on the latest `master` undoes all the formatting changes in this file for me. Do you happen to remember the steps to repro? Did you just do a Git commit, and a hook (husky/pretty-quick?) changed the file for you?

Reply:
@friederbluemle, agreed. i was surprised at these changes myself!  just the scripts is what did it on my box!